UPGRADE-5.3.md

```diff
@@ -0,0 +1,12 @@
+UPGRADE FROM 5.2 to 5.3
+=======================
+
+FrameworkBundle
+---------------
+
+ * Deprecated the `x-forwarded-all` option. Use a combination of `x-forwarded-*` options instead.
+
+HttpFoundation
+--------------
+
+ * Deprecated the `Request::HEADER_X_FORWARDED_ALL` constant. Use a combination of `HEADER_X_FORWARDED_*` constants instead.
```

Comment:
Can you add more description, like you did in the PR? Something like this:

```
Use one the following instead:

    Request::setTrustedProxies(['1.2.3.4'], Request::HEADER_X_FORWARDED_FOR | Request::HEADER_X_FORWARDED_HOST | Request::HEADER_X_FORWARDED_PORT | Request::HEADER_X_FORWARDED_PROTO);
    Request::setTrustedProxies(['1.2.3.4'], Request::HEADER_X_FORWARDED_TRAEFIK);
    framework.trusted_headers: [x-forwarded-for, x-forwarded-host, x-forwarded-port, x-forwarded-proto]
```

And probably copy/paste it in other changelog files.

Reply:
done